Add link to self-managed backend on self-hosted pages. (#110)
pulumi/pulumi-hugo: themes/default/content/docs/guides/self-hosted/api.md

---

{{% notes type="info" %}}

> This page is for Self-Hosting of the **Pulumi Enterprise Edition**. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
> 
> For the **open source self-managed backends** see [State and Backends]({{< relref "/docs/intro/concepts/state" >}}).

Reviewer: ```suggestion
Self-hosting is only available with **Pulumi Enterprise Edition**. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.

To manage your state with a self-managed backend, see [State and Backends]({{< relref "/docs/intro/concepts/state" >}}).
```
